ygnmi/ygnmi.go

```diff
@@ -727,21 +735,24 @@ func NewBatch[T any](root SingletonQuery[T]) *Batch[T] {
 }
 
 // AddPaths adds the paths to the batch. Paths must be children of the root.
-func (b *Batch[T]) AddPaths(paths ...PathStruct) error {
+func (b *Batch[T]) AddPaths(paths ...UntypedQuery) error {
```

Comment:
this is backwards incompatible, we should probably just add a func instead of changing this one

Reply:
Discussed offline, I think it's ok to have this breaking change since it is a relatively easy fix for downstream code. I will document how to it in the release notes.